src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="../src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="card__image"
+      >
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="card__code"><span>Product code: 195434</span></div>
```

Comment:
in comment about span/semantic tag (preferred) meant _replace_ div tag instead of adding wrapper:
```suggestion
      <p class="card__code">Product code: 195434</p>
```